frontend/src/app/main/ui/workspace/sidebar/options/menus/grid_cell.cljs

```diff
@@ -65,7 +65,7 @@
                         :id     (dm/str "align-self-end-" type)}]
 
       [:& radio-button {:value "stretch"
-                        :icon  (if is-col? i/align-self-row-strech i/align-self-column-strech)
+                        :icon  (if is-col? i/align-self-row-stretch-refactor i/align-self-column-stretch-refactor)
```

Comment:
I think this is swapped (column and row)
![image](https://github.com/penpot/penpot/assets/1483050/4f24ce50-d26a-44df-afa2-b7043bcb728f)

Reply:
fixed